### PR TITLE
[Form] Documented the choice_self_translation option of LanguageType

### LINKED ISSUE #12745: [Form] Allow to translate each language into its language in LanguageTy…
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#32388
| PR author(s) | @javiereguiluz
| Merged in    | 5.1-dev

### PR DESCRIPTION
**Pull request by javiereguiluz**
Fixes #12745.

### CODE REVIEW

File: reference/forms/types/language.rst [hunk 1]
```diff
@@ -74,6 +75,19 @@ If this option is ``true``, the choice values use the `ISO 639-2 alpha-3`_
 three-letter codes (e.g. French = ``fra``) instead of the default
 `ISO 639-1 alpha-2`_ two-letter codes (e.g. French = ``fr``).
 
+choice_self_translation
```
Comment (xabbuh) [inline reference/forms/types/language.rst]: By only looking at the option name I would not really know what this is supposed to do. Can we think of something more self-explanatory?

Comment (javiereguiluz) [reply reference/forms/types/language.rst]: I agree with you. I've updated the description. Is it more clear now? Thanks.

Comment (xabbuh) [reply reference/forms/types/language.rst]: I was rather think about renaming the option. But sadly I cannot come up with a better name. :(

**Review by OskarStark — APPROVED** [review]

File: reference/forms/types/language.rst (hunk 1 above)
Comment (xabbuh) [reply reference/forms/types/language.rst]: But we can discuss that separately.

**Review by xabbuh — APPROVED** [review]

**Pull request merged**